packages/base/Page/src/PageArticle/PageArticle.tsx

```diff
@@ -1,34 +1,25 @@
 import type { ReactNode, HTMLAttributes } from 'react'
 import React, { forwardRef } from 'react'
-import type { Theme } from '@material-ui/core/styles'
-import { makeStyles } from '@material-ui/core/styles'
-import cx from 'classnames'
 import type { BaseProps, OverridableComponent } from '@toptal/picasso-shared'
 import { Container } from '@toptal/picasso-container'
-
-import styles from './styles'
+import { twMerge } from '@toptal/picasso-tailwind-merge'
 
 export interface Props extends BaseProps, HTMLAttributes<HTMLDivElement> {
   /** Custom components that render content of page */
   children: ReactNode
 }
 
-const useStyles = makeStyles<Theme>(styles, {
-  name: 'PicassoPageArticle',
-})
-
 export const PageArticle: OverridableComponent<Props> = forwardRef<
   HTMLDivElement,
   Props
 >(function PageArticle(props, ref) {
   const { children, className, style, ...rest } = props
-  const classes = useStyles()
 
   return (
     <Container
       {...rest}
       ref={ref}
-      className={cx(classes.root, className)}
+      className={twMerge('flex-1 my-0 mx-4 md:mx-8 lg:mx-8 xl:mx-8', className)}
```

Comment:
```suggestion
      className={twMerge('flex-1 my-0 mx-4 md:mx-8', className)}
```

Tailwind follows mobile first, so this should be enough